src/components/avatar/avatar.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import oneLineTrim from 'common-tags/lib/oneLineTrim';
+import styles from './avatar.mod.css';
+import filterDataAttributes from '../../utils/filter-data-attributes';
+
+const getFirstChar = str =>
+  typeof str === 'string'
+    ? str
+        .trim()
+        .slice(0, 1)
+        .toUpperCase()
+    : '';
+
+const getInitialsFromName = ({ firstName = '', lastName = '' }) =>
+  `${getFirstChar(firstName)}${getFirstChar(lastName)}`;
+
+/**
+ * `s` - defines the size. We want a bigger one if the user is on a retina-display
+ * `d` - defines the default if the user is not known to Gravatar. It returns a blank image,
+ *        which let the initials underneath shine through
+ *
+ * @see: https://de.gravatar.com/site/implement/images/
+ */
+const createGravatarImgUrl = (md5Hash, size) =>
+  `https://www.gravatar.com/avatar/${md5Hash}?s=${size}&d=blank`;
+
+const getSizeInPx = size => do {
+  if (size === 'l') 100;
+  else if (size === 'm') 48;
+  else 26;
+};
+
+const GravatarImg = props => (
+  <img
+    className={styles['gravatar-img']}
+    src={createGravatarImgUrl(props.hash, getSizeInPx(props.size))}
+    srcSet={oneLineTrim`
```

Comment:
`srcset` is usually spelled in lowercase. I think we should use lowercase as well, unless this is a React thing (which I don't think it is).